src/test/lib/utils/meta_tables/meta_temporary_memory_usage_table_test.cpp

```diff
@@ -0,0 +1,110 @@
+#include "base_test.hpp"
+
+#include "hyrise.hpp"
+#include "storage/table.hpp"
+#include "utils/meta_tables/meta_temporary_memory_usage_table.hpp"
+
+namespace opossum {
+
+class MetaTemporaryMemoryUsageTest : public BaseTest {
+ protected:
+  std::shared_ptr<AbstractMetaTable> _meta_temporary_memory_usage_table;
+
+  void SetUp() override {
+    _meta_temporary_memory_usage_table = std::make_shared<MetaTemporaryMemoryUsageTable>();
+  }
+
+  void TearDown() override {
+    Hyrise::reset();
```

Comment:
`Hyrise::reset()` is called by the `BaseTest` destructor, no need to call it again.